README.md

```diff
@@ -67,7 +67,7 @@ Installation instructions for previous releases can be found in [Metrics Server
 
 Metrics Server | Metrics API group/version | Supported Kubernetes version
 ---------------|---------------------------|-----------------------------
-0.6.x          | `metrics.k8s.io/v1beta1`  | 1.19+
+0.6.x          | `metrics.k8s.io/v1beta1`  | 1.19-1.24
```

Comment:
As discussed in issue https://github.com/kubernetes-sigs/metrics-server/issues/1037. this update is correct